jupyterhub/app.py

```diff
@@ -316,7 +316,7 @@ def _load_classes(self):
 
     @validate("config_file")
     def _validate_config_file(self, proposal):
-        if not os.path.isfile(proposal.value):
+        if not self.generate_config and not os.path.isfile(proposal.value):
```

Comment:
I've considered if it would make sense to do something specific on generate_config and a file exist, but that is handled already by asking to overwrite or not at a later stage, so if generate_config is true, the file can both exist already and not.

Reply:
So, this LGTMyself ;)